doc/src/cylc-user-guide/suiterc.tex

```diff
@@ -754,16 +754,99 @@ \subsection{[scheduling]}
 the time zone determined by \ref{cycle-point-time-zone} if that is set.
 
 \begin{myitemize}
-    \item {\em type:} ISO 8601 date-time point representation (e.g.\ 
+    \item {\em type:} ISO 8601 date-time point representation (e.g.\
  \lstinline=CCYYMMDDThhmm=, 19951231T0630) or ``now''.
     \item {\em default:} (none)
 \end{myitemize}
 
 The string ``now'' converts to the current date-time on the suite host (adjusted
 to UTC if the suite is in UTC mode but the host is not) to minute resolution.
 Minutes (or hours, etc.) may be ignored depending on your cycle point format
+
 (\ref{cycle-point-format}).
 
+\paragraph[initial cycle point relative to current time]{[scheduling] \textrightarrow [[initial cycle point]] \textrightarrow initial cycle point relative to current time}
+
+This can be used to set the initial cycle point time relative to the current time.
+
+Two additional commands, \lstinline=next= and \lstinline=previous=, can be used when setting the initial cycle point.
+
+The syntax uses truncated ISO8601 time representations, and is of the style: \lstinline=next(Thh:mmZ)=, \lstinline=previous(T-mm)=; e.g.
```

Comment:
I think `previous(T-mm)` was used here to show that it can be either `Thh:mmZ` or `T-mm`... but distinguishing between `next` and `previous` at the same time could be confusing? I had to read the next examples to make sure I could use `Thh:mmZ` with `previous`.

So maybe instead give three examples? `next(Thh:mmZ)`, `next(T-mm)`, `previous(Thh:mmZ)`? Or even the include the other one? Not sure if being too picky here, feel free to resolve this conversation if the other examples are sufficient :+1:

Reply:
I did wonder about that, but I thought that the table of examples would be enough to demonstrate that all the specifications work with both next and previous.

Reply:
Righto, agreed!